scipy-1.0/paper.tex

```diff
@@ -950,10 +944,10 @@ \subsection*{Statistical distributions}
 distributions and a framework to implement additional distributions. Given an
 expression for a probability distribution function (pdf), the framework will
```

Comment:
Is any of this new? Consider removal except for the new stuff at the end.

Reply:
See #97 for this discussion -- maybe do that there

Reply:
ping @chrisb83

Reply:
I think the framework to handle distributions is worth mentioning, but let's discuss in 97